.circleci/continue_config.yml

```diff
@@ -0,0 +1,278 @@
+  golang-validate-all:
+    shell: /bin/bash -eo pipefail
+    executor:
+      name: go/default # is based on cimg/go
+      tag: '1.21'
+    steps:
+      - checkout
+      - install-just
+      - install-foundry
+      - run:
+          name: run validation checks on all chains
+          command: just validate "*"
+      - notify-failures-on-main:
+          channel: C07GQQZDW1G # to slack channel `notify-superchain-validation-failures`
+  golang-promotion-test:
+    shell: /bin/bash -eo pipefail
+    executor:
+      name: go/default # is based on cimg/go
+      tag: '1.21'
+    steps:
+      - checkout
+      - install-just
+      - install-foundry
+      - run:
+          name: run validation checks
+          command: just promotion-test
+      - slack/notify:
+          event: always
+          custom: |
+            {
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": ":control_knobs: The daily report on standard candidate chains is ready!"
+                  },
+                  "accessory": {
+                    "type": "button",
+                    "text": {
+                      "type": "plain_text",
+                      "text": "👀 View Report",
+                      "emoji": true
+                    },
+                    "url": "${CIRCLE_BUILD_URL}"
+                  }
+                },
+                {
+                  "type": "divider"
+                },
+                {
+                  "type": "context",
+                  "elements": [
+                    {
+                      "type": "mrkdwn",
+                      "text": "Click the link to see what validation checks would fail if all candidate chains were promoted to standard (and exclusions removed)."
+                    }
+                  ]
+                }
+              ]
+            }
+          channel: C07GZVCCUS0 # to slack channel `notify-superchain-promotion-reports`
+  publish-bot:
+    environment:
+      NODE_AUTH_TOKEN: $NPM_TOKEN # Use NPM_TOKEN as the auth token
+    docker:
+      - image: cimg/node:18 # Use Node.js 18
+    steps:
+      - checkout
+      - run:
+          name: Set deployment token
+          command: npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
+      - env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run:
+          name: "Build and publish package on NPM \U0001F4E6"
+          command: pnpm release
+  check-codegen:
+    executor:
+      name: go/default # is based on cimg/go
+      tag: '1.21'
+    steps:
+      - checkout
+      - install-just
+      - run:
+          name: Run codegen
+          command: just codegen
+      - run:
+          name: check git tree is clean
+          command: git diff --exit-code
+  golang-validate-genesis-allocs:
+    docker:
+      - image: cimg/go:1.22.6-node
```

Comment:
Will using go v1.22.6 cause any issues since the go.mod/go.work files specify v1.21?

Reply:
Good catch -- I don't think it will cause issues but I will adjust this if I can to make sure.